test/test-helper-functions.js

```diff
@@ -6,7 +6,7 @@
      * an external JavaScript file.
      *
      * @param {string} src The `src` of the external JavaScript file.
```

Comment:
Shouldn't it be ``String`` instead of ``string`` (also in the following 5 changesets)?